msg/estimator_status.msg

```diff
@@ -9,6 +9,8 @@ float32[3] vibe			# IMU vibration metrics in the following array locations
 
 float32[24] covariances	# Diagonal Elements of Covariance Matrix
 
+float32[3] output_tracking_error # return a vector containing the output predictor angular, velocity and position tracking error magnitudes (rad), (m/s), (m)
```

Comment:
Can you split this up into 3 self-documenting fields? Having this as a vector makes things opaque.

Reply:
Agree that this would be clearer. These message changes affect some log analytics in the CI. I prefer to make it not more complicated for now.

Reply:
I don't get it. This is a new field you're adding in this PR. How is this already used in / affect analytics?

Reply:
I moved it from the ekf2_innovations message to the the estimator_status message

Reply:
Can you address my original comment?

Reply:
The output_tracking_error is a vector that is constructed in ECL EKF. It contains different metrics similar to the estimator_status.vibe field, and the different components are described well in the comments. Since we would need to change things in a lot of different places, I would prefer leaving it like this.